addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml

```diff
@@ -1,794 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-<!--
-    Copyright (C) 2008-2008 凯源吕鑫 lvxin@gmail.com   <basic chart data>
-    Copyright (C) 2008-now  开阖软件 jeff@osbzr.com    < PM and LTS >
-
-    会计科目表模板
--->
-
     <data noupdate="1">
-
-        <record id="chart10003" model="account.account.template">
+        <record id="account_1003" model="account.account.template">
             <field name="name">转让帐户</field>
             <field name="code">1003</field>
-            <field name="reconcile" eval="True"/>
-            <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+            <field name="reconcile" eval="True" />
+            <field name="user_type_id" ref="account.data_account_type_current_assets" />
         </record>
-
         <!-- Chart template -->
-        <record id="l10n_chart_china_standard_business" model="account.chart.template">
-            <field name="name">中国会计科目表  （财会[2006]3号《企业会计准则》</field>
-            <field name="code_digits" eval="4"/>
-            <field name="currency_id" ref="base.CNY"/>
+        <record id="l10n_cn_standard" model="account.chart.template">
```

Comment:
Changing xmlid of such objects is not possible in stable version. Please keep the previous one or I will must merge it in master only.